transport_nantes/mailing_list/views.py

```diff
@@ -15,7 +15,7 @@
 from django.urls import reverse_lazy
 from django.utils.crypto import get_random_string
 from django.views.generic import FormView, ListView, TemplateView
-from topicblog.models import TopicBlogEmailSendRecord
+from topicblog.models import SendRecordMarketingEmail
```

Comment:
People can subscribe, unsubscribe, resubscribe, unsubscribe.  So this is a transactional mail, not a marketing mail.  Sending it to you once doesn't end the cycle.